dist/src/main/resources/application.properties

```diff
@@ -37,6 +37,5 @@ management.endpoint.loggers.enabled=true
 # Elastic client which spawns 16 threads)
 spring.autoconfigure.exclude=\
   org.springframework.boot.autoconfigure.elasticsearch.ElasticsearchRestClientAutoConfiguration, \
-  org.springframework.boot.autoconfigure.netty.NettyAutoConfiguration, \
```

Comment:
:question: If I understand correctly, removing this from the excludes is not strictly necessary for the metrics but it gives Spring the option to enable/disable netty leak detection? Just to make sure I understand correctly.

Reply:
It doesn't enable anything by default, but it'll let us modify the leak detection via Spring config and not just via system properties. As we're likely to look into this, I figured it would be helpful :+1: We can keep it excluded and configure via System properties though, but I prefer using the same configuration as we do everywhere else. Happy to remove this if you'd rather though, it's not that important.

Reply:
No that makes sense, let's keep it :+1: